projects/groceries/submissions.csv

```diff
@@ -1,3 +1,3 @@
 github_username, repository_url
-my_user, https://github.com/my_user/my_repo
+Cynthia0524, https://github.com/Cynthia0524/nyu-info-2335-70-201706/projects/groceries/groceries.py
```

Comment:
@Cynthia0524 Sorry if the instructions weren't clear, but please create a separate GitHub repository (not your fork of the course repository) to contain your work, and then submit the URL to that repository here next time you submit.

Also there are a lot of merge conflicts in this PR, so please make sure to refresh your fork with `git reset --hard upstream/master` before submitting next time (see https://github.com/prof-rossetti/nyu-info-2335-70-201706/blob/master/CONTRIBUTING.md#updating-your-fork).